wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsViewController.swift

```diff
@@ -324,6 +337,66 @@ extension GroupDetailsViewController {
 
 }
 
+// MARK: - Conversation verification status
+
+private extension GroupDetailsViewController {
+
+    var verificationStatus: NSAttributedString? {
```

Comment:
Creating an attributed string is redundant. The `GroupConversationVerificationStatusView` should be used.